netcam_rtsp.c

```diff
@@ -935,14 +945,17 @@ int netcam_setup_rtsp(netcam_context_ptr netcam, struct url_t *url){
 */
 int netcam_next_rtsp(unsigned char *image , netcam_context_ptr netcam){
```

Comment:
With the elimination of the rtsp components in this function and the new information regarding what thread is calling this, should we move this function out of the netcam_rtsp.c module and over to the netcam.c?  It also now appears that we do not need the #IF on ffmpeg anymore.